react/features/base/react/components/native/styles.js

```diff
@@ -0,0 +1,201 @@
+import {
```

Comment:
What should be added to this file versus other styles.js files?

Reply:
Yeah, nothing really, see previous comment :)